datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -235,9 +255,9 @@ CREATE INDEX ON dcsa_im_v3_0.document_party (shipping_instruction_id);
 DROP TABLE IF EXISTS dcsa_im_v3_0.displayed_address CASCADE;
 CREATE TABLE dcsa_im_v3_0.displayed_address (
     -- Same key as document_party
-    party_id uuid NOT NULL REFERENCES dcsa_im_v3_0.party (id),
+    party_id varchar(100) NOT NULL REFERENCES dcsa_im_v3_0.party (id),
     shipment_id uuid NULL REFERENCES dcsa_im_v3_0.shipment (id),
-    shipping_instruction_id uuid NULL REFERENCES dcsa_im_v3_0.shipping_instruction (id),
+    shipping_instruction_id varchar(100) NULL REFERENCES dcsa_im_v3_0.shipping_instruction (id),
     party_function varchar(3) NOT NULL REFERENCES dcsa_im_v3_0.party_function (party_function_code),
     address_line varchar(250) NOT NULL,
     address_line_number int NOT NULL
```

Comment:
This is not correct - it must be a reference to an entity containing address lines since it is an array??